ludwig/datasets/mnist/__init__.py

```diff
@@ -58,8 +57,11 @@ def process_downloaded_dataset(self):
         os.makedirs(self.processed_temp_path, exist_ok=True)
         for dataset in ["training", "testing"]:
             print(f'>>> create ludwig formatted {dataset} data')
-            labels, data = self.read_source_dataset(dataset, self.raw_dataset_path)
-            self.write_output_dataset(labels, data, os.path.join(self.processed_temp_path, dataset))
+            labels, data = self.read_source_dataset(dataset,
```

Comment:
do we need to do any checks with both labels and/or data before calling write_output_dataset?

Reply:
Probably not, @ANarayan could tell you more